frappe/utils/data.py

```diff
@@ -197,67 +197,68 @@ def nowtime():
 	"""return current time in hh:mm"""
 	return now_datetime().strftime(TIME_FORMAT)
 
-def get_first_day(dt, d_years=0, d_months=0, as_str=False):
```

Comment:
While I fully agree with the requirement for this change, changing argument names will certainly break someone's function call that's using named arguments. E.g. `get_first_day(dt=some_val)` will result in "unkown argument dt"

Reply:
Right! I will do that, thanks!

Reply:
Done!